webui/react/src/pages/WorkspaceList.tsx

```diff
@@ -58,7 +58,7 @@ const WorkspaceList: React.FC = () => {
   const [pageError, setPageError] = useState<Error>();
   const [isLoading, setIsLoading] = useState(true);
   const pageRef = useRef<HTMLElement>(null);
-  const [canceler] = useState(new AbortController());
+  const canceler = useRef<AbortController | null>(null);
```

Comment:
**nit:** this is probably just a preference thing:
```
const canceler = useRef<AbortController>();
```
`canceler` naturally resolves to `undefined | AbortController`, without an explicit `null` declaration.